platform/feature-api/communication/postoffice/README.md

```diff
@@ -58,10 +58,10 @@ module, including via HTTP or `MessagePort`s.
 On the other side, clients can obtain a callable object:
 
 ```typescript
-import {endpointClient, ClientOf} from "@polypoly-eu/postoffice";
-import {client} from "@polypoly-eu/port-authority";
+import { backendClient, ClientOf } from "@polypoly-eu/postoffice";
```

Comment:
Again, redundant. A client needs to be a client of the backend.

Reply:
Was also previously redundant, as far as a _renaming_ is concerned I find this OK. Note that there's already `client`, that'd need some wider renaming.